Snowflake.Data/Logger/EasyLoggerManager.cs

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.IO;
+using log4net;
+using log4net.Appender;
+using log4net.Layout;
+using Snowflake.Data.Configuration;
+
+namespace Snowflake.Data.Log
+{
+    internal class EasyLoggerManager
+    {
+        private static EasyLoggerManager s_instance = null;
+
+        private static readonly object s_instanceLock = new object();
+        private static readonly object s_lock = new object();
+
+        public static EasyLoggerManager Instance
+        {
+            get
+            {
+                lock (s_instanceLock)
+                {
+                    if (s_instance == null)
+                    {
+                        s_instance = new EasyLoggerManager();
+                    }
+
+                    return s_instance;
+                }
+            }
+        }
+        private const string AppenderPrefix = "SFEasyLogging";
+        
+
+        private readonly EasyLoggingLevelMapper _levelMapper = EasyLoggingLevelMapper.Instance;
+        
+        public virtual void ReconfigureEasyLogging(EasyLoggingLogLevel easyLoggingLogLevel, string logsPath)
+        {
+            var log4netLevel = _levelMapper.ToLog4NetLevel(easyLoggingLogLevel);
+            lock (s_lock)
+            {
+                var repository = (log4net.Repository.Hierarchy.Hierarchy)LogManager.GetRepository();
+                var rootLogger = (log4net.Repository.Hierarchy.Logger)repository.GetLogger("Snowflake.Data");
+                rootLogger.Level = log4netLevel;
+                var appender = logsPath == "STDOUT"
```

Comment:
Do we support only this two types of appenders?

Reply:
yes. That how it was done in jdbc driver.